### PR TITLE
Show reserve page ready, Im working in delete function

### PR DESCRIPTION
**Pull request by yvallejos**

### CODE REVIEW

File: app/components/reserves/edit/showReserve.controller.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    activate();
+
+    function activate() {
+        EditReserveService.getReserve(vm.reserveId)
+        .then(createReserve); 
+    }
+
+    function createReserve(reserve){
+      console.log("entre a createReserve");
+      console.log(reserve);
```
Comment (Emmurillo) [inline app/components/reserves/edit/showReserve.controller.js]: Unnecessary logs.

File: app/components/reserves/edit/showReserve.controller.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      EditReserveService.getUser(reserve.user_ref)
+      .then(
+        function(user){
+          vm.user = user;
```
Comment (Emmurillo) [inline app/components/reserves/edit/showReserve.controller.js]: So far I don't see any usage for this object, there's no need to bind it to the controller. Let me know if I'm missing something.

**Comment by Emmurillo** [comment]
@yvallejos please open a different PR for the delete feature. PRs should only handle one concern at the time.

File: app/components/reserves/edit/showReserve.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            <span>{{vm.reserve.timestamp | date:' dd-MM-yyyy '}}</span>
+          </div>
+        </div>
+        <br><br>
+        <div class="form-group">
+          <label class="col-md-3 control-label"></label>
+          <div class="col-md-8">
+            <button type="button" class="btn btn-danger" ng-click="vm.deleteReserve()">Delete</button>
+            <span></span>
+            <input class="btn btn-default" value="Editar" type="reset">
```
Comment (Emmurillo) [inline app/components/reserves/edit/showReserve.html]: Why input? It's inconsistent with the `Delete` button.

File: app/components/reserves/edit/showReserve.module.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('app.showReserve', [
```
Comment (Emmurillo) [inline app/components/reserves/edit/showReserve.module.js]: Wasn't this module already registered?

Comment (yvallejos) [inline app/components/reserves/edit/showReserve.module.js]: @Emmurillo this module is for a specific reserve the other one is for all the reserves

File: app/components/reserves/edit/showReserve.controller.js (hunk 2 above)
Comment (yvallejos) [inline app/components/reserves/edit/showReserve.controller.js]: @Emmurillo  with the user object I get the name of the user in the reserve

**Comment by yvallejos** [comment]
show

**Comment by johnnyxu-pernix** [comment]
@yvallejos please resolve the conflicts, and also change the commit message, since you're not going to push anything related to the delete function.

**Comment by Emmurillo** [comment]
@yvallejos as per our conversation in the chatroom, you should follow the next standard for the files:
```
reserves/
      reserves.module.js
      reserves.route.js
      reserves.service.js
      add/
             reserves.add.html
             reserves.add.controller.js
      list/
             reserves.list.html
             reserves.list.controller.js
      show/
….
      edit/
….
```

Then the only module you need to register should be inside the `reserves.module.js` file, and therefore use that module in any other related `js`  file (router, service, controller, etc) inside the `reserves` folder

File: app/components/reserves/show/reserve.show.controller.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      reservesService.getUser(reserve.user_ref)
+      .then(
+        function(user){
+          vm.user = user;
+          vm.reserve = { 'user_name': user.contact, 'timestamp': reserve.timestamp, 'reserveId': reserve.reservation_number};
+        });
+    }
+
+    function deleteReserve(){
+          console.log("si elimino");
```
Comment (johnnyxu-pernix) [inline app/components/reserves/show/reserve.show.controller.js]: Delete this console log

File: app/components/reserves/show/reserve.show.controller.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    function editReserve(){
+          console.log("editar");
```
Comment (johnnyxu-pernix) [inline app/components/reserves/show/reserve.show.controller.js]: Same here

File: app/styles/reserveShow.css [hunk 7]
```diff
@@ -0,0 +1,3 @@
+.reserve-button-container{
+  margin-top:  2em;
+}
```
Comment (johnnyxu-pernix) [inline app/styles/reserveShow.css]: Add a new line here

File: app/components/users/list/users.list.controller.js [hunk 8]
```diff
@@ -2,18 +2,18 @@
...
 
   angular
-    .module('app.showUsers')
-    .controller('ShowUsersController', showUsersController);
+    .module('app.users')
+    .controller('ListUsersController', listUsersController);
 
   /* @ngInject */
-  function showUsersController(ShowUserService) {
+  function listUsersController(usersService) {
```
Comment (johnnyxu-pernix) [inline app/components/users/list/users.list.controller.js]: The controller should start with uppercase

**Comment by johnnyxu-pernix** [comment]
@Emmurillo do you think that we should name the controllers `NameCtrl` like you did with the nav bar, or continue with `NameController`?

**Pull request merged**